### PR TITLE
NEEDED_PLUS_ONES should be integer

### PR DESCRIPTION
**Pull request by paveldruzyak**
Just found that we receive string and it should be converted to integer.
Also I ignore `0`, but not sure about this.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -31,8 +31,8 @@
   @payload = JSON.parse(payload_body)
 
   # If $required_plus_ones has been specified, change the default
-  if params['required_plus_ones']
-    NEEDED_PLUS_ONES = params['required_plus_ones']
+  if params['required_plus_ones'].to_i > 0
```
Comment (jstruzik) [inline app.rb]: Casting to an integer here will cause `nil` to have a `0` value no matter what.

Could do something like:

```ruby
if params['required_plus_ones'] && params['required_plus_ones'].to_i > 0
```

Comment (paveldruzyak) [inline app.rb]: @jstruzik yech, but we should ignore `nil` or not?

Comment (jstruzik) [inline app.rb]: I think if a client passes in `null` as a `POST` param or if it doesn't pass in the parameter at all, we should ignore it completely.

Comment (paveldruzyak) [inline app.rb]: I mean here is `>` not `>=`, so `nil` we be ignored anyway

Comment (jstruzik) [inline app.rb]: Ah yeah, valid point.

**Comment by jstruzik** [comment]
:+1: Ty for this fix.

**Comment by Rican7** [comment]
:+1:

**Pull request merged**